makefile.shade

```diff
@@ -610,18 +609,15 @@ var CAN_BUILD_ONECORE = '${Directory.Exists(VS_ONECORE_ROOT) && Directory.GetFil
                     extraBinaries.Add("System.IO.FileSystem.Watcher.dll");
                     extraBinaries.Add("System.IO.MemoryMappedFiles.dll");
                     extraBinaries.Add("System.IO.Pipes.dll");
-                    extraBinaries.Add("System.Net.Security.dll");
+                    extraBinaries.Add("System.Net.Http.dll");
                     extraBinaries.Add("System.Runtime.Extensions.dll");
-                    extraBinaries.Add("System.Security.Cryptography.Encoding.dll");
-                    extraBinaries.Add("System.Security.Cryptography.Primitives.dll");
                     extraBinaries.Add("System.Security.Cryptography.Algorithms.dll");
                     extraBinaries.Add("System.Security.SecureString.dll");
                     extraBinaries.Add("System.Text.Encoding.CodePages.dll");
                     extraBinaries.Add("System.Threading.dll");
```

Comment:
What's the rule for us to decide what assemblies are required here? How do we know when to change?